filters/fadein/fadein_test.go

```diff
@@ -431,7 +431,7 @@ func TestPostProcessor(t *testing.T) {
 				if !ep.Detected.After(firstDetected) {
 					t.Fatal("Failed to reset detection time.")
 				}
-				if endpointRegistry.GetMetrics(ep.Host).DetectedTime().After(firstDetected) {
+				if !endpointRegistry.GetMetrics(ep.Host).DetectedTime().After(firstDetected) {
 					t.Fatal("Failed to reset detection time.")
```

Comment:
Does the message hold true after the logical flip?

Reply:
Actually, this message was not properly formulated before this PR, I was expecting endpointregistry **not to reset** detection time (to have **different** behavior compared to fadeIn postprocessor), but didn't mentioned it in this message explicitly.
However, after discussion[1] with Alexander we decided that endpointregistry **must reset** detection time too (to have **the same** behaviour as fadeIn postprocessor). And therefore this poorly formulated message becomes true.

[1] https://github.com/zalando/skipper/pull/2795#discussion_r1427804501